src/module/meme-object.js

```diff
@@ -5,7 +5,13 @@ const setLocal = () => localStorage.setItem('memelist', JSON.stringify(memeList)
 // === get local storage === //
 const getLocal = () => JSON.parse(localStorage.getItem('memelist'));
 // ==================== //
-
+// === update the index values when delete an item === //
+const updateindex = () => {
```

Comment:
- [ ] After editing the task description the changes should also save to local storage so if we refresh the page we should get that edited description. In your project editing task description is not updating the local storage so when I refresh the page I get the old description on the page. Please fix this issue.

### See the image for reference

![image](https://user-images.githubusercontent.com/83803414/160412462-b2d4a9ff-c2d8-4fce-a973-8df609267b78.png)